<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandDock</class>
 <widget class="QDockWidget" name="BandDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Band</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="toolTip">
         <string>update viewports to reflect labeling changes</string>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
        <property name="icon">
         <iconset resource="../gerbil.qrc">
          <normaloff>:/toolbar/left</normaloff>:/toolbar/left</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="markerSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>label to edit</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToMinimumContentsLength</enum>
        </property>
        <property name="minimumContentsLength">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="toolTip">
         <string>clear current label or seeds</string>
        </property>
        <property name="icon">
         <iconset theme="edit-clear">
          <normaloff/>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="alphaSlider">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>labeling opacity</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="singleStep">
         <number>3</number>
        </property>
        <property name="pageStep">
         <number>25</number>
        </property>
        <property name="value">
         <number>63</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="toolTip">
         <string>labeling opacity</string>
        </property>
        <property name="text">
         <string>Î±</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="screenshotButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>save image to file</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-save">
          <normaloff/>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="AutohideView" name="view">
      <property name="baseSize">
       <size>
        <width>250</width>
        <height>300</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutohideView</class>
   <extends>QGraphicsView</extends>
   <header>widgets/autohideview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>markerSelector</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>alphaSlider</tabstop>
  <tabstop>applyButton</tabstop>
 </tabstops>
 <resources>
  <include location="../gerbil.qrc"/>
 </resources>
 <connections/>
</ui>
